spec/features/idv/account_creation_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+feature 'account creation after LOA3 request', idv_job: true do
+  include SamlAuthHelper
+  include IdvHelper
+
+  context 'successful IdV with same phone as 2FA' do
+    it_behaves_like 'idv account creation', 'saml'
```

Comment:
No particular reason. It's just the first thing that came to my mind. I can make it a symbol.